common/build/uboot.mk

```diff
@@ -47,6 +47,18 @@ $(error If TARGET_UBOOT_ENV is set TARGET_UBOOT_ENV_SIZE must also be set. See\
 endif
 endif
 
+# Check if we can sign u-boot image
+ifneq ($(CST_BIN),)
+ifneq ($(SIGN_KEY),)
+ifneq ($(IMG_KEY),)
+ifneq ($(SRK_TABLE),)
```

Comment:
Please have it has a one-liner, something like:
```
ifneq ($(BLAH)$(BLAH)$(BLAH)$(BLAH),)
endif
```

Reply:
All variables must be defined to enable the signing of u-boot. In a one-liner statement, the condition will be true if one of them is empty.
I'm not sure my proposition is the most elegant one but I don't know how to do better for now.
What do you think about it ?

Reply:
Can you try:
```
ifneq ($(and $(VAR1),$(VAR2),$(VAR3)),)
```
Is this working?
credits: stackoverflow ;)

Reply:
How I didn't see it?
It work perfectly!